github-actions/index.html

```diff
@@ -0,0 +1,158 @@
+<!doctype html>
+<html class="no-js" lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>University of Illinois GitHub Service Community Portal: GitHub Actions</title>
+  <meta name="description" content="Community-supported documentation for the University of Illinois GitHub service.">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="robots" content="noindex, nofollow">
+
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+    integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+  <link rel="styleheet" href="/css/illinois-system.css">
+
+  <meta name="theme-color" content="#fafafa">
+</head>
+
+<body>
+  <header class="blog-header my-3 pb-3 border-bottom">
+    <div class="container">
+      <h1>University of Illinois GitHub Service Community Portal - GitHub Actions</h1>
```

Comment:
I think putting the service portal name first obscures the purpose of this page. My vote would be to reverse the order ("Github Actions - University ...") or to use the service portal name only as a breadcrumb in the title element (`<title>Github Actions - University of Illinois...</title>`")

Reply:
Ok, I went ahead and pulled out the "University ..." out and added breadcrumbs